src/DetailsView/components/reflow-assessment-view.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { ContentLinkDeps } from 'views/content/content-link';
+
+import { AssessmentTestResult } from '../../common/assessment/assessment-test-result';
+import { Tab } from '../../common/itab';
+import {
+    AssessmentData,
+    AssessmentNavState,
+    PersistedTabInfo,
+} from '../../common/types/store-data/assessment-result-data';
+import { GettingStartedView } from './getting-started-view';
+import { TargetChangeDialog, TargetChangeDialogDeps } from './target-change-dialog';
+
+export type WithAssessmentTestResult = { assessmentTestResult: AssessmentTestResult };
```

Comment:
where is this being used?

Reply:
AssessmentTestResult is used to pass the getting started information for each assessment

Reply:
removing it!